tests_e2e/test_e2e.py

```diff
@@ -12,8 +12,12 @@
 
 @pytest.fixture(scope="module") 
 def driver():
- #   with webdriver.Firefox(executable_path='/Users/bacis/Documents/Corndel/Module-3/DevOps-Course-Starter/geckodriver') as driver:
-    with webdriver.Firefox() as driver:
+ #   with webdriver.Firefox() as driver:
```

Comment:
Delete commented out code. It lives on in git in case you really do want it.

Reply:
done. yes bad habit